test/loader_test.rb

```diff
@@ -45,6 +45,15 @@
         CoffeeShop::Controller.configuration.default_response_format.must_equal(:html)
       end
 
+      it 'configures controller to use default headers' do
+        CoffeeShop::Controller.configuration.default_headers.
+          must_equal({
+            "X-Frame-Options" => "DENY",
+            "Content-Security-Policy" => "form-action 'self'; referrer origin-when-cross-origin; reflected-xss block; manifest-src 'self'; frame-ancestors 'self'; base-uri 'self'; default-src 'none'; connect-src 'self'; img-src 'self'; style-src 'self'; font-src 'self'; object-src 'self'; plugin-types application/pdf application/x-shockwave-flash application/x-java-applet; child-src 'self'; media-src 'self'",
```

Comment:
@joneslee85 Same of https://github.com/hanami/hanami/pull/504/files#r53554599 :wink: